### ISSUE TITLE
Access to meeting recording for scheduled group meetings

### ISSUE THREAD

**Issue by bodekf** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#7957]
Greetings,
This page does not cover one important scenario, that we just discovered the hard way today. When a meeting invite is sent to a group associated with a team, the only people who are given access to view the meeting recording afterwards are those who actually attended the meeting. For some reason, it does not add the entire group by name to the access control list of the recording, but rather only adds names of people from within the group that were present in the meeting. This was reported by one of our profs and I was able to reproduce it.
Bodek

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 976b8a9b-5a3e-1f3b-b848-d1dfbfd3389d
* Version Independent ID: 2b13a471-3baf-0086-439a-f94abb162fd6
* Content: [Teams cloud meeting recording - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/cloud-recording)
* Content Source: [Teams/cloud-recording.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/cloud-recording.md)
* Service: **msteams**
* GitHub Login: @tonysmit
* Microsoft Alias: **tonysmit**

**Comment by scanum** [comment]
Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by dariomws** [comment]
Hi @bodekf, thank you for your feedback.

Are you sure that this case is not included in the following rows?
![image](https://user-images.githubusercontent.com/33589238/136015632-07eb9574-85c1-4cf2-8a62-f1b729e22983.png)

Specially this sentence: "All other **meeting members** have read access without ability to download."

**Comment by bodekf** [comment]
I am sure. We reproduced the issue. All other users do NOT have read access; only users who were present in the meeting have read access. Users who did not attend the meeting has no access. Steps to reproduce:


  1.  Compose the invite in Outlook calendar and place a group/team that has several members in Required field.
  2.  Start the meeting, have some but not all members of the group/team join the meeting
  3.  Start recording
  4.  Stop recording
  5.  Examine the access control of the video recording. Note that group/team name was not added; instead you will see only names of people who were in attendance.

**Comment by dariomws** [comment]
@bodekf I believe you. What I'm saying is that "meeting members" are the people who joined the call.

**Comment by bodekf** [comment]
I see what you’re saying now, but this wording is not clear enough. If I misunderstood its intended meaning, chances are other users will as well. I recommend that you reword it to make it more explicit.

Also, “members of the meeting” is not the best term for this. I think “meeting attendees” would make more sense. So you could say something like “members of the group/team invited to the meeting who were in attendance of the meeting have read access without the ability to download. Members of the group who did not attend the meeting will have no access.

Finally, you need to break it up into two separate cases. There is a different outcome when the invite is sent to people listing their names individually vs when it is sent to a group. In first case, all attendees that were invited will have read access to the recording, even those who were not in attendance of the meeting. Not the case when the invite is sent to a group.

**Comment by dariomws** [comment]
@bodekf my apologies but I couldn't reproduce it.

I sent a Teams meeting to an M365 group which is also a team. I joined the meeting as an organizer and with an additional attendee. Start a recording and stop it after a few seconds. The attendee that joined the meeting could see the recording but another one which didn't join the meeting could also see it.
One thing to pay attention to is the chat window. There, I saw that all the members of the group (the people which received the invite) were added to the chat ("<organizerName> added <attendee1> and X other to the chat").

I cannot provide more help here as we are limited to documentation issues and improvements and I don't have enough evidence to update the article.

I think the best way forward is if you open a service ticket in your tenant so this can get resolved ASAP. Based on the outcome, let me know if it can be called out in the docs.

Please follow this link to contact support for business products:  https://docs.microsoft.com/office365/admin/contact-support-for-business-products.

Please keep us posted here on the resolution to feed whatever information you discover into the content.

**Comment by dariomws** [comment]
Thank you @bodekf!
